scripts/getReleaseStatus.js

```diff
@@ -10,14 +10,15 @@ const endpoints = [
       DEV: 'https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk/info',
     },
   },
+  { application: 'activities', urlEnv: 'ACTIVITIES_URL' },
```

Comment:
Any reason not to go with separate envs like for adjudications?

Reply:
This is using the url from the ACTIVITIES_URL env var so will have env specificity built in